Bursary content added to visa page (#3323)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

    Salaries and benefits of teaching in England : "/salaries-and-benefits"
    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
promo_content:

navigation: 20.10
navigation_title: Fees and financial support for non-UK trainee teachers

> navigation_description: Learn more about teacher training fees and help from the UK government for physics and languages trainees.

Reviewer: ```suggestion
navigation_description: Learn more about teacher training fees in England and financial help for physics and languages trainees.
```
